src/celeritas/ext/detail/GeantProcessImporter.cc

```diff
@@ -224,25 +224,14 @@ void append_table(G4PhysicsTable const* g4table,
     switch (table_type)
     {
         case ImportTableType::dedx:
-        case ImportTableType::dedx_process:
```

Comment:
It's possible we may use the per-process energy loss tables in the future (#253), but we can add it back in if needed.

Reply:
If we do that, I think it'd make more sense to still just call it `dedx` (and omit the `range` at that point) since the data is associated with a single process.